cmd/installer/subcommands/daemon/run_windows_test.go

```diff
@@ -8,21 +8,79 @@
 package daemon
 
 import (
+	"github.com/DataDog/datadog-agent/pkg/util/filesystem"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/fx"
+	"os"
+	"strings"
 	"testing"
 
 	"github.com/DataDog/datadog-agent/cmd/installer/command"
-	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
-	"github.com/spf13/cobra"
 )
 
-func TestRunCommand(t *testing.T) {
-	cmd := runCommand(&command.GlobalParams{})
-	cmd.GroupID = ""
-	t.Skipf("Since runFxWrapper no longer calls fxutil.OneShot directly, but instead svc.Run, which blocks, " +
-		"the test cannot be performed. Skip it for now.")
-	fxutil.TestOneShotSubcommand(t,
-		[]*cobra.Command{cmd},
-		[]string{"run"},
-		run,
-		func() {})
+const (
+	datadogYaml = "C:\\ProgramData\\Datadog\\datadog.yaml"
```

Comment:
unit tests shouldn't modify the system. Accessing/modifying this file will fail unless the test is run as Admin.

Can we create a tempfile and provide the path to the config component instead?

Alternatively, we can run the test as an integration test
https://github.com/DataDog/datadog-agent/blob/d1ef19abd517f009a13ac710f8bda6d4f4330a88/tasks/agent.py#L570